pom.xml

```diff
@@ -81,6 +81,9 @@
     <RMM_LOGGING_LEVEL>OFF</RMM_LOGGING_LEVEL>
     <SPARK_RAPIDS_JNI_CXX_FLAGS/>
     <USE_GDS>OFF</USE_GDS>
+    <LIBCUDF_BUILD_TYPE>Release</LIBCUDF_BUILD_TYPE>
+    <BUILD_TYPE>${LIBCUDF_BUILD_TYPE}</BUILD_TYPE>
+    <CUDA_FLAGS_DEBUG>-g</CUDA_FLAGS_DEBUG>
```

Comment:
Is this intended to be overridden by the user for some use cases?  I assume this is only used for Debug builds, and I'm a bit surprised this needs to be specified at all.

Reply:
Yeah, it's a little weird, but the default flags only adds `-g` (which enables debugging in host code), you need `-G` to enable debugging in device code as well.